### PR TITLE
Mobility Profile

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
## Description

Works with https://github.com/ibi-group/otp-middleware/pull/200.

This PR includes changes for enabling in config the mobility profile pages in the settings pages,
whether it is on creating new accounts or existing accounts.

To enable mobility profiles, simply add a line in config.yml with:
```
mobilityProfile: true
```

Behind the scenes, the code for `UserAccountScreen` is heavily modified, and the "wizard" concept (previously `SequentialPaneDisplay`) is used more widely for the mobility profile setup screen.

**PR Checklist:**
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by AdrianaCeric — APPROVED** [review]
Thanks for the changes! As a next step, it is not clear the settings the user selected from the main settings page. It would be great that after parameters are defined that they are shown on the main page like this:
image
`Define Your Mobility Profile` would be changed to `Edit Your Mobility Profile` after the user has edited it >0 times

File: lib/components/user/mobility-profile/mobility-wizard.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  const title = intl.formatMessage({
+    id: 'components.MobilityProfile.title'
+  })
+
+  return (
+    <Wizard
+      activePaneId={activePaneId}
+      formikProps={formikProps}
+      pages={['mobility1', 'mobility2']}
```
Comment (amy-corson-ibigroup) [inline lib/components/user/mobility-profile/mobility-wizard.tsx:23]: Can we change these to "assistive devices" and "limitations"?

File: lib/components/user/wizard.tsx [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        // (e.g. save a user account).
+        if (onNext) {
+          await onNext()
+        }
+        this._routeTo(nextId)
+      }
+      this._focusHeader()
+    } else {
+      // Display a toast to acknowledge saved changes
+      // (although in reality, changes quietly took effect in previous screens).
```
Comment (amy-corson-ibigroup) [inline lib/components/user/wizard.tsx:101]: Oh, cool!

**Review by amy-corson-ibigroup — APPROVED** [review]
Looks really great! I do agree with Adriana's comment - ideally users should be able to see their selected settings without entering the wizard. Overall though, looks really good and just a couple of small questions

**Comment by binh-dam-ibigroup** [comment]
> `Define Your Mobility Profile` would be changed to `Edit Your Mobility Profile` after the user has edited it >0 times

Updated in https://github.com/opentripplanner/otp-react-redux/pull/1090/commits/d6808558857321a2eb5633d3daea9746622f92e9

File: lib/components/user/mobility-profile/mobility-wizard.tsx (hunk 1 above)
Comment (binh-dam-ibigroup) [reply lib/components/user/mobility-profile/mobility-wizard.tsx]: Yes, I did in https://github.com/opentripplanner/otp-react-redux/pull/1090/commits/b1ad318057f5692261dcef584e2f0bc224f0abb5.

File: lib/components/user/mobility-profile/assistive-devices-pane.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  'crutches',
+  'stroller',
+  'service-animal',
+  'mobility-scooter',
+  'electric-wheelchair',
+  'manual-wheelchair'
+]
+
+const DeviceOption = styled.span`
+  display: block;
```
[the remark this answers is not in the document]
Comment (binh-dam-ibigroup) [reply lib/components/user/mobility-profile/assistive-devices-pane.tsx]: Addressed in https://github.com/opentripplanner/otp-react-redux/pull/1090/commits/23edbc895f522f2f3c66c07f8a42a20f1b0665d6.

**Pull request merged**